### ISSUE TITLE
Using the light theme is breaking the search function for me

### ISSUE THREAD

**Issue by gamemaniac999** [deep-gaurav/MusicPiped#115]
I'm on version 2.1.7 and android 8. The other day the search just stopped working for some reason so I tried everything I could think from reinstalling to rebooting my phone. Nothing was working until earlier on a whim I switched to dark theme and it worked again.
![Screenshot_20191127-161028](https://user-images.githubusercontent.com/58269443/69762910-42cca900-1131-11ea-8238-de48d98e9504.png)
![Screenshot_20191127-161041](https://user-images.githubusercontent.com/58269443/69762911-42cca900-1131-11ea-9fb4-f2ec552e6409.png)

**Comment by Emonshr** [comment]
Same here.

**Comment by 0x4a** [comment]
Yes. Was going to comment that search is broken, but in fact works fine in dark mode...

**Comment by deep-gaurav** [comment]
This is for play store version or FDroid version?

**Comment by 0x4a** [comment]
F-Droid for me
